src/inc/msquicp.h

```diff
@@ -134,6 +134,10 @@ typedef struct QUIC_PRIVATE_TRANSPORT_PARAMETER {
 #define QUIC_PARAM_CONN_KEEP_ALIVE_PADDING              0x85000003  // uint16_t
 #define QUIC_PARAM_CONN_DISABLE_VNE_TP_GENERATION       0x85000004  // BOOLEAN
 
+#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
+#define QUIC_PARAM_STREAM_RELIABLE_OFFSET_RECV          0x08000006  // uint64_t
```

Comment:
```suggestion
#define QUIC_PARAM_STREAM_RELIABLE_OFFSET_RECV          0x88000000  // uint64_t
```
Private ones have the high bit set (i.e. the `8` at the beginning). And it's a different number space from non-private, so the first one here starts with 0 at the low bits.